src/extractors/items.js

```diff
@@ -13,13 +13,63 @@ module.exports = ({ items }, outputDirectory) => new Promise((resolve, reject) =
   for (let name in items.item) {
     const item = items.item[name]
 
-    const itemData = {
+    let itemData = {
       id: item.numeric_id,
       displayName: item.display_name,
       name: item.text_id,
       stackSize: item.max_stack_size
     }
 
+    // Manual fixes
```

Comment:
Good but a better way to do this would be reading a manual_items_override.json file.
That can be for a next commit